docs/docs/07-forms.md

```diff
@@ -65,7 +70,9 @@ In this example, we are simply accepting the newest value provided by the user a
   }
 ```
 
-This would accept user input but truncate the value to the first 140 characters.
+This would accept user input and truncate the value to the first 140 characters.
+
+**Controlled**, for inputs or otherwise, means it doesn't maintain its own internal state; it renders based on properties.
```

Comment:
This sentence parses funny to me.  Perhaps we change it to something like:

 > Saying that a component is **Controlled** means it doesn't maintain its own internal state; the component renders purely based on props.